packages/cron-utils/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
```

Comment:
Maybe empty this file?

Reply:
Removed this file. release-please will create one from scratch if it doesn't exist